### PR TITLE
Create Migration

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- [#27916](https://edu-redmine.sun-asterisk.vn/issues/27916)

## WHAT this PR does?
- Create Migration

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+/public/bower_components/*
```
Comment (huyenttm-0901) [inline .gitignore]: end of file

**Review by huyenttm-0901 — COMMENTED** [review]
cmt 1 tý, em sửa luôn đi nhé

**Comment by sondth-1037** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
(merged)

**Pull request merged**